templates/machine.swift.motemplate

```diff
@@ -79,7 +79,7 @@ class _<$managedObjectClassName$>: <$customSuperentity$> {
 <$foreach Relationship noninheritedRelationships do$>
 <$if Relationship.isToMany$>
     @NSManaged
-    var <$Relationship.name$>: <$Relationship.immutableCollectionClassName$>
+    var <$Relationship.name$>: Set<<$Relationship.destinationEntity.managedObjectClassName$>>
```

Comment:
I've got a very similar pull request up that preserves ordered relationships. https://github.com/rentzsch/mogenerator/pull/275